cfme/tests/containers/test_static_custom_attributes.py

```diff
@@ -131,14 +131,12 @@ def test_add_attribute_with_empty_name(provider):
         * You should get an error
         * You should not see this attribute in the custom  attributes table
     """
-    try:
+    with pytest.raises(APIException):
         provider.add_custom_attributes(
             CustomAttribute('', "17")
         )
         pytest.fail('You have added custom attribute with empty name'
```

Comment:
This will take some verification but... I think the pytest.fail isn't necessary anymore.  If the exception isn't raised the test will fail with a message like: `Failed: Expecting APIException`.

Did you test locally and find that this pytest.fail was evaluated on no raised exception?

Doesn't look like raises() takes an option for a failure message if the exception isn't raised.